tailwind.config.js

```diff
@@ -28,15 +24,7 @@ module.exports = {
         1: '1rem',
         6: '1.5rem',
       },
-      maxWidth: {
-        fc: 'fit-content',
-      },
-      minHeight: {
-        50: '50vh',
-        80: '80vh',
-      },
       backgroundImage: {
-        hero: 'url(/hero-pattern.svg)',
         texture: 'url(/texture.svg)',
```

Comment:
```suggestion
        
```
this also can be removed since this is also being used dynamically [here](https://github.com/hussamkhatib/scriptified.dev/blob/main/pages/index.tsx#L27) like `hero-pattern` is?

Reply:
resolved now